orderbook/src/database/events.rs

```diff
@@ -15,18 +16,31 @@ pub struct EventIndex {
 
 #[derive(Debug)]
 pub enum Event {
-    Trade(Trade),
+    DbTrade(DbTrade),
     Invalidation(Invalidation),
 }
 
-#[derive(Debug, Default)]
-pub struct Trade {
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash)]
```

Comment:
Do we really need all these added derivations? It seems to me like this file should actually not be touched by this PR (execpt for the rename in the imports).

Reply:
Yea possibly, this is remnants from having moved it and put it back. I'll check it out.